app/representers/metric_links_representer.rb

```diff
@@ -1,15 +1,17 @@
+# frozen_string_literal: true
+
 module MetricLinksRepresenter
   extend ActiveSupport::Concern
-
   included do
 
     link :metric do
       return unless metric
 
       if metric.method_metric?
-        admin_api_service_metric_method_url(metric.service, metric.parent, metric)
+        admin_api_service_metric_method_url(metric.service, metric.parent, metric.id)
```

Comment:
Aren't method also enabled on backend_api level too?

Reply:
Yes, but in https://github.com/3scale/porta/pull/1201 because originally I had it all in this PR and it was over 1000 lines so I decided to divide the task in 2 PRs 😄